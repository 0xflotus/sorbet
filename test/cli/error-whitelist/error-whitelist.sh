#!/bin/bash
main/sorbet --silence-dev-message test/cli/error-whitelist/error-whitelist.rb --error-white-list=7017 2>&1
