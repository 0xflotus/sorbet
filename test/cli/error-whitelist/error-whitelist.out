test/cli/error-whitelist/error-whitelist.rb:6: This function does not have a `sig` http://go/e/7017
     6 | def foo
         ^^^^^^^
  Autocorrect: Use `-a` to autocorrect
    test/cli/error-whitelist/error-whitelist.rb:6: Insert `sig {generated.returns(Integer)}
 `
     6 | def foo
         ^
  Autocorrect: Use `-a` to autocorrect
    test/cli/error-whitelist/error-whitelist.rb:6: Insert `  extend T::Helpers
`
     6 | def foo
        ^
Errors: 3
